### PR TITLE
adding auth part to app

### PR DESCRIPTION
**Pull request by rajaranjanmr**
adding auth part to app

### CODE REVIEW

**Review by sri-rishi — APPROVED** [review]
Code looks good , just a remainder remove `console` and commented code from files when you are done working on that features

File: src/components/Navigation.js [hunk 1]
```diff
@@ -31,13 +33,19 @@ function Navigation() {
...
+
           <li>
             <Link to="/login">Login</Link>
           </li>
-          
+}
+{isLoggedIn &&
+
           <li>
             <Link to="/logout">Logout</Link>
```
Comment (nikhiljugale007) [inline src/components/Navigation.js]: suggestion: you can use navlinks here

File: src/components/video-page-category.js [hunk 2]
```diff
@@ -142,15 +147,15 @@ function VideoListingPage(props) {
           <div className="like">
             <i
               class="fa fa-thumbs-up"
-              onClick={() => likedClickHandler(props.value)}
+              onClick={() => isLoggedIn ? likedClickHandler(props.value):navigate("/login")}
```
Comment (nikhiljugale007) [inline src/components/video-page-category.js:151]: suggestion: try to move onclick event to separate function and call the function here

Comment (nikhiljugale007) [inline src/components/video-page-category.js:151]: suggestion: you can destructor props at top instead of doing props.values

**Review by nikhiljugale007 — COMMENTED** [review]
added few suggestion, rest code looks good. try to remove console.log.

**Pull request merged**